app/components/configure-minted-asset/ConfigureMintedAsset.tsx

```
@@ -0,0 +1,180 @@
+          </p>
+        </button>
+
+        <Tabs index={tabIndex} onChange={handleTabsChange}>
+          <TabList className="my-2 gap-1">
+            {tabList.length > 0 &&
+              tabList.map((label, i) => (
+                <Tab
+                  key={i}
+                  name={label}
+                  disabled={noActiveClaim(label)}
+                  className={`min-w-12 rounded-sm px-4 py-2 ${noActiveClaim(label) ? `` : `hover:bg-slate-400`} ${noActiveClaim(label) ? `` : ` hover:text-slate-800 `}${label === activeTab ? `text-slate-800` : `text-slate-500`} ${noActiveClaim(label) ? `hover:cursor-not-allowed` : `hover:cursor-pointer`} ${
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)